pkg/models/elasticSubnetConfig.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package models
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+type ElasticSubnetConfig struct {
```

Comment:
This raises some interesting questions about persistence. I think for now we should delete this when you clean. Later, we might want to save it so you can easily redeploy it or move from local to fuji to mainnet.